DTS-STN/Alpha-Site
# Add validation for minimum age

# Description

[signup form should check for a minimum user age](https://trello.com/c/zBZb288a)

Set minimum age check on the year of birth field. Will check that the year entered is current not greater than 18 years ago.

![image](https://user-images.githubusercontent.com/34345435/121586451-ac559500-ca01-11eb-92df-20290ee635ee.png)

![image](https://user-images.githubusercontent.com/34345435/121586488-b6779380-ca01-11eb-93b8-6dfe5d8094d7.png)


There may also be some cleanup with gitignore and the pre-commit hook here, which probably should have gone into it's own PR.

## Acceptance Criteria

The user must be at least 18 to sign up. There is validation on the year of birth field.

## Test Instructions

1. Navigate to the user signup form
2. Enter a year of birth less than 18 years ago
3. Try to submit the form
4. See the error about needing to be at least 18


## Checklist


## Product and Sprint Backlog

[Trello](https://trello.com/b/ZqWtJSyh/alpha-site-board)
